SSI.php

```diff
@@ -160,8 +160,8 @@
 
 	if (!isset($_SESSION['session_value']))
 	{
-		$_SESSION['session_var'] = substr(md5(mt_rand() . session_id() . mt_rand()), 0, rand(7, 12));
-		$_SESSION['session_value'] = md5(session_id() . mt_rand());
+		$_SESSION['session_var'] = substr(md5(random_int(0, PHP_INT_MAX) . session_id() . random_int(0, PHP_INT_MAX)), 0, rand(7, 12));
+		$_SESSION['session_value'] = md5(session_id() . random_int(0, PHP_INT_MAX));
```

Comment:
I would keep `random_int()` for setting the session var and value. They aren't, strictly speaking, cryptographic values, but the point is to make a temporary and unique identifier for the user to prevent impersonation, so the more random, the better.